modules/social_features/social_follow_user/social_follow_user.module

```diff
@@ -187,3 +211,73 @@ function social_follow_user_preprocess_flag(array &$variables): void {
     }
   }
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter() for user_form().
+ */
+function social_follow_user_form_user_form_alter(array &$form, FormStateInterface $form_state): void {
+  $profile = _social_profile_get_profile_from_route();
+  $active = \Drupal::config('social_follow_user.settings')->get('status');
+
+  if ($profile instanceof ProfileInterface && $active) {
+    $user_data = \Drupal::service('user.data');
+    $disable_following = $user_data->get('social_profile_privacy', (int) $profile->get('uid')->target_id, 'disable_following');
```

Comment:
@volodymyr-sydor I would suggest to rename it to `following_enabled`
 or `following_enabled`.

Double negatives are bad in plain English as well as computer code. See explanation [here](https://stackoverflow.com/questions/13470724/naming-convention-isenable-vs-isdisable).

Reply:
I agree but then using it for this label seems doesn't match:
<img width="902" alt="Screenshot 2022-06-03 at 09 23 54" src="https://user-images.githubusercontent.com/25609390/171798888-b8ef9658-3475-4abf-9bd4-8dbbe272cd1a.png">

Reply:
done